packages/ai-lab/src/components/AILabVideo/AILabLocalVideo.tsx

```diff
@@ -186,6 +197,20 @@ export const AILabLocalVideo = ({ perf, perfCallback, src }: VideoProps) => {
     setPerfProps(undefined);
   }
 
+  useEffect(() => {
+    setInterval(
+      () =>
+        console.log(
+          `Current Time: ${videoRef.current?.currentTime} - ${
+            videoRef.current?.duration
```

Comment:
when we're calculating FPS, shouldn't we calculate inferences frames, rather than video frames?

Reply:
I think a good way to do this would be that on each inference, you would do a check against a `performance.now()` and indicate the FPS of the model vs the video.